server/src/main/java/io/druid/server/coordinator/CoordinatorDynamicConfig.java

```diff
@@ -202,10 +216,11 @@ public int hashCode()
     private boolean emitBalancingStats;
     private int balancerComputeThreads;
     private Set<String> killDataSourceWhitelist;
+    private double balancerSlop;
 
     public Builder()
     {
-      this(15 * 60 * 1000L, 524288000L, 100, 5, 15, 10, 1, false, null);
+      this(15 * 60 * 1000L, 524288000L, 100, 5, 15, 10, 1, false, null, 0.001F);
```

Comment:
any clue on how this number was chosen? Is it an absolute or relative value?

I think we should preserve the existing behavior by default, since most production deployments are constantly adding new data anyway, in which case it doesn't necessarily make sense to stop rebalancing. Also, even a small number may cause very large clusters to completely stop rebalancing because the incremental improvement is typically very small.